solidity/dashboard/src/components/DelegationOverview.jsx

```diff
@@ -50,6 +50,17 @@ const DelegationOverview = ({
   return (
     <section>
       <DelegationHeader type={context} selectedGrant={selectedGrant} />
+      <LoadingOverlay
+        isFetching={areTopUpsFetching}
+        skeletonComponent={<DataTableSkeleton columns={3} />}
+      >
+        <Tile>
+          <TopUpsDataTable
+            topUps={filteredTopUps}
+            initializationPeriod={initializationPeriod}
+          />
+        </Tile>
+      </LoadingOverlay>
```

Comment:
One thing I want to consider (cc @liz-shinn) is whether we should also move existing delegations to the top (though not a huge deal).